SPECS/openssh.spec

```diff
@@ -653,6 +651,9 @@ getent passwd sshd >/dev/null || \
 %endif
 
 %changelog
+* Tue Apr 30 2024 Thierry Escande <thierry.escande@vates.tech> - 7.4p1-23.2.2 + 0.10.3-2
```

Comment:
So, there are two versions in one spec file, so both should be incremented when we change the packaging.

We could define `%xcpng_subrel` and add it to each `Release` tag.

Reply:
Done